### PR TITLE
enhance: Add precheck when chunk manager init

### PR DESCRIPTION
**Pull request by zhagnlu**
#28329

### CODE REVIEW

File: internal/core/src/storage/MinioChunkManager.cpp [hunk 1]
```diff
@@ -216,6 +216,26 @@ MinioChunkManager::BuildS3Client(
     }
 }
 
+void
+MinioChunkManager::PreCheck(const StorageConfig& config) {
+    LOG_SEGCORE_INFO_ << "start to precheck chunk manager with configuration:"
+                      << config.ToString();
+    try {
+        ListWithPrefix();
```
Comment (xige-16) [inline internal/core/src/storage/MinioChunkManager.cpp:224]: List all the objects in the bucket, isn't it a bit dangerous?

Comment (zhagnlu) [reply internal/core/src/storage/MinioChunkManager.cpp:224]: yes. I consider it,  maybe cost a little. but no find safe and little-cost solution.

Comment (zhagnlu) [reply internal/core/src/storage/MinioChunkManager.cpp:224]: another solution, give a prefix that not exist, just test connection not real result

**Comment by zhagnlu** [comment]
rerun ut

File: internal/core/src/storage/MinioChunkManager.cpp (hunk 1 above)
Comment (xiaofan-luan) [reply internal/core/src/storage/MinioChunkManager.cpp]: should we do a simple exist fucntion for "/"

Comment (zhagnlu) [reply internal/core/src/storage/MinioChunkManager.cpp]: I think it has a similar performance effect as list not exists prefix

**Pull request merged**